src/pages/Missions.js

```diff
@@ -1,23 +1,77 @@
+import '../css/mission.css';
 import React, { useEffect } from 'react';
-import axios from 'axios';
-import { useDispatch } from 'react-redux';
-import setMission from '../redux/mission/MissionAction';
-import MissionCard from '../components/MissionCard';
+import Table from 'react-bootstrap/Table';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { useDispatch, useSelector } from 'react-redux';
+import { addremoveMembertoMission, LoadMessions } from '../redux/mission/MissionAction';
 
 const Missions = () => {
-  const dispatch = useDispatch();
   const fetchMissions = async () => {
-    const response = await axios
-      .get('https://api.spacexdata.com/v3/missions');
-    dispatch(setMission(response.data));
+    const missions = [];
+    await fetch('https://api.spacexdata.com/v3/missions')
+      .then((res) => res.json())
+      .then((json) => {
+        json.forEach((e) => {
+          const mission = {
+            mission_id: e.mission_id,
+            mission_name: e.mission_name,
+            description: e.description,
+            isMember: false,
+          };
+          missions.push(mission);
+        });
+      });
+    return missions;
   };
+  const dispatch = useDispatch();
+  useEffect(() => { fetchMissions().then((res) => dispatch(LoadMessions(res))); }, []);
+  const { error, loading, missions } = useSelector((state) => state.mission);
+  const Missions = missions;
 
-  useEffect(() => {
-    fetchMissions();
-  }, []);
+  console.log(error, loading, Missions);
```

Comment:
While console.log statements are great for debugging, kindly endeavour to remove them afterwards in order to clean up the code base